connaisseur/validators/cosign/cosign_validator.py

```diff
@@ -186,6 +186,15 @@ def __get_cosign_validated_digests(self, image: str, trust_root: dict):
                 image=str(image),
                 trust_root=trust_root["name"],
             )
+        elif "Error: entity not found in registry\nmain.go:" in stderr:
```

Comment:
Since this list seems to grow, maybe it would (now/soon) be time to put this error parsing into a separate function

Reply:
good point. will create an issue

Reply:
#584